tests/ut/test_decorators.py

```diff
@@ -559,21 +491,18 @@ async def what(self, keys=None, what=1):
         assert str(inspect.signature(what)) == "(self, keys=None, what=1)"
         assert inspect.getfullargspec(what.__wrapped__).args == ["self", "keys", "what"]
 
-    async def test_reuses_cache_instance(self):
-        with patch("aiocache.decorators._get_cache", autospec=True) as get_c:
-            cache = create_autospec(AbstractBaseCache, instance=True)
-            cache.multi_get.return_value = [None]
-            get_c.side_effect = [cache, None]
+    async def test_reuses_cache_instance(self, mock_cache):
+        # TODO @review can probably just remove this test.
```

Comment:
As noted, this test is a bit superfluous now.

Reply:
Yeah, drop it.